packages/main/dev/vite.config.js

```diff
@@ -14,6 +14,7 @@ export default defineConfig({
     emptyOutDir: false,
     minify: isProduction ? 'esbuild' : false,
     sourcemap: false,
+    target: 'es2019', // portal-ui cannot handle the nullish coalescing output by ESNext
```

Comment:
is this a reversion?  i don't know enough about `es` versions.

Reply:
I am not sure what you mean by reversion - the Vite [docs](https://vitejs.dev/config/build-options.html#build-target) say the default `target` is 'modules' which is effectively `es2020`

Reply:
So it is going backwards in a way but it should not have an effect on the functionality.

I think it is possible that perhaps there will be new syntax in future `es` targets that we would not want to be transpiled away / might not be possible to use under an earlier `target`. For this reason, we should still track the status of the portal-ui bundling setup to potentially remove or bump the `target` in the future, and we can let the portal-ui developers know

Reply:
Ok, I'm fine with that.  As long as they know. I'm sure upgrading to v5 would have benefits.